python/lsst/dm/OCPS/ocps_csc.py

```diff
@@ -36,10 +37,13 @@
 $schema: http://json-schema.org/draft-07/schema#
 $id: https://github.com/lsst/dm_OCPS/blob/master/schema/OCPS.yaml
 # title must end with one or more spaces followed by the schema version, which must begin with "v"
-title: OCPS v2
+title: OCPS v3
 description: Schema for OCPS configuration files
 type: object
 properties:
+  instance:
+    description: Name of OCPS instance this configuration is for; checked against index enum
```

Comment:
Yes, the value needs to be a name in the enum.

All this may change with the new configuration proposal, but for now I still want to be able to change configurations at `start` time, but only to configurations that make sense for the index.  I didn't really want a separate configuration repo for each index, although I guess that might be a possibility.  This seemed like the next best thing, with configuration names as the first line of defense and a check of a value actually inside the configuration to make sure.

I'll add more to the description.

Reply:
Thanks. I think the design is fine; I just wanted more explanation.

I agree with you about having a separate config repo for each index of OCPS.

Reply:
(It is technically possible to write a configuration that can be applied to multiple instances, but I think it will be unusual.)